string logger_name
string severity_threshold
---
bool success
